scripts/studyarea_to_WKT/studyarea_to_WKT.yml

```diff
@@ -0,0 +1,31 @@
+script: studyarea_to_WKT.R
+name: study area geometry to WKT
+description: This script converts the geometry of the study area into a well-known text (WKT) representation.
+author: 
+ - name: Maria Alejandra Molina Berbeo (mmolina@humboldt.org.co)
+ - name: Victor Julio Rincon
+external_link: https://github.com/GEO-BON/biab-2.0
```

Comment:
Link to a place where we can find the geopackages (and remove the gpkg from the repo)